.github/workflows/centos7-ci.yml

```diff
@@ -29,6 +29,15 @@ jobs:
       with:
         submodules: recursive
 
+    - name: Build rpm package
+      run: |
+        sudo gem install --no-document fpm
+        # WOULD CHANGE TO MASTER WHEN PR GETS MERGED
+        git clone https://github.com/idbeta/build-tools.git
+        cd build-tools
+        git checkout new-method-to-package
```

Comment:
We can use `-b` to specify the branch
Like

```
git clone -b new-method-to-package  https://github.com/idbeta/build-tools.git
```

Reply:
Thanks! Fixed